### PR TITLE
fix(K8s): Updated Recommended alert policy doc 

### PR DESCRIPTION
**Pull request by nbaenam**
Updated the [Recommended alert policy](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policy/) doc according to the UI.

Ticket: NR-317064

### CODE REVIEW

File: src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx [hunk 1]
```diff
@@ -122,7 +122,7 @@ These partners require specific login and deployment procedures for you to use y
     2. Follow [New Relic's instructions](/docs/agents/php-agent/installation/wordpress-users-new-relic) to complete the installation process, or contact [W3TC support](http://wordpress.org/support/plugin/w3-total-cache) for assistance.
     3. Wait for New Relic to begin collecting data.
 
-       Within a few minutes after deploying the agent, performance data about your app will appear on APM's [Applications Overview p](/docs/apm/applications-menu/monitoring/applications-overview-dashboard)age.
+       Within a few minutes after deploying the agent, performance data about your app will appear on APM's [Applications Overview](/docs/apm/applications-menu/monitoring/applications-overview-dashboard) page.
```
Comment (akristen) [inline src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx:125]:
```suggestion
       Within a few minutes after deploying the agent, performance data about your app will appear on the APM [Applications Overview](/docs/apm/applications-menu/monitoring/applications-overview-dashboard) page.
```

Comment (akristen) [reply src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx:125]: We don't make New Relic possessive, so thought we might avoid it with our capabilities. this is very nitpicky and not a blocker tho!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx [hunk 2]
```diff
@@ -53,7 +53,7 @@ Take a look at what you can do with New Relic:
 
 * Scrape Prometheus metrics from any workload in the cluster thanks to the service discovery provided by the [Prometheus Agent](/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#Agent).
 
-* The integration includes a set of [predefined alert conditions](/docs/kubernetes-pixie/kubernetes-integration/installation/predefined-alert-policy), but you can [create and modify your Kubernetes alerts](/docs/kubernetes-pixie/kubernetes-integration/installation/create-alerts) based on your Kubernetes data, or add a set of [recommended alert policy](/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policy). 
+* The integration includes a set of [predefined alert conditions](/docs/kubernetes-pixie/kubernetes-integration/installation/predefined-alert-policy), but you can [create and modify your Kubernetes alerts](/docs/kubernetes-pixie/kubernetes-integration/installation/create-alerts) based on your Kubernetes data, or add a set of [recommended alert policy](/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies).
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx:56]:
```suggestion
* The integration includes a set of [predefined alert conditions](/docs/kubernetes-pixie/kubernetes-integration/installation/predefined-alert-policy), but you can [create and modify your Kubernetes alerts](/docs/kubernetes-pixie/kubernetes-integration/installation/create-alerts) based on your Kubernetes data, or add a set of [recommended alert policies](/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies). 
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies.mdx [hunk 3]
```diff
@@ -11,59 +11,93 @@ freshnessValidatedDate: 2024-09-02
...
 
-While we've tried to address the most common use cases in all environments, there are a number of additional alerts you can set up to extend the default policy. These are our recommended alerting policies.
+While we've tried to address the most common use cases in all environments, there are a number of additional alerts you can set up to extend the default policy. See [Getting started with New Relic alerts](/docs/tutorial-create-alerts/create-new-relic-alerts/) to know more about alerts.
 
-## Adding the recommended alert policy [#add-recommended-alert-policy]
+## Adding the recommended alert conditions and dashboards [#add-recommended-alert-policy]
 
-To add recommended alert policy, follow these steps:
+To add recommended alert policy and dashboards, follow these steps:
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies.mdx:20]:
```suggestion
To add recommended alert policies and dashboards, follow these steps:
```

**Review by akristen — DISMISSED** [review]
Found some nitpicky stuff, but those can easily be fast follows :)

File: src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx:125]: Good catch! Modified!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx:56]: Changed!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/recommended-alert-policies.mdx:20]: Changed!

**Review by akristen — APPROVED** [review]

**Pull request merged**